http-core/src/main/resources/reference.conf

```diff
@@ -307,6 +307,17 @@ pekko.http {
       # Fail the connection if a sent ping is not acknowledged within this timeout.
       # When zero the ping-interval is used, if set the value must be evenly divisible by less than or equal to the ping-interval.
       ping-timeout = 0s
+
+      frame-type-throttle {
+        # Configure the throttle for non-data frame types (https://github.com/apache/incubator-pekko-http/issues/332)
+        # supported frame-types for throttlng are:
+        # reset, headers, continuation, go-away, priority, ping, push-promise, window-update
```

Comment:
While we are here, should we documented a suggested list of frame types and/or should we create an alias for all of them?

Reply:
I'd prefer not to provide too much guidance. If users want to protect against CVE-2023-44487, they just need `frame-types = ["reset"]`. In @jrudolph's original solution, almost the full list in line 314 was throttled (the exception was `window-update` wasn't throttled but there was a TODO to work out whether it should be throttled).

If we do make a recommendation, I would suggest `frame-types = ["reset"] and suggest to users that they might want to expand the list.

Reply:
@mdedetrich wdyt?

Reply:
I think we can just keep this simple, a user can add what they want.